### ISSUE TITLE
Global config targets that are common to all groups

### ISSUE THREAD

**Issue by aviaviavi** [aviaviavi/confetti#2]

**Comment by dmarcoux** [comment]
This could be achieved by having a top level key `common`, like this:

```
common:
  - ~/.abc
  - ~/.def
groups:
  - name: aws
    targets:
      - ~/.aws/credentials
  - name: dotfiles
    targets:
      - ~/.spacemacs
    search_paths:
      - path: ~/dotfiles
        recursive: true
```

**Comment by aviaviavi** [comment]
@dmarcoux I like it. While we're at it, I'd probably take it one small step further so we could do something like:
```
common:
  targets:
    - ~/.abc
    - ~/.def
  search_paths:
    - path: ~/a
      recursive: false
```

**Comment by aviaviavi** [comment]
resolved with version 0.3
